packages/devtools_app/lib/src/screens/memory/panes/tracing/tracing_pane_controller.dart

```diff
@@ -8,10 +8,15 @@ import 'package:flutter/widgets.dart';
 import 'package:vm_service/vm_service.dart';
 
 import '../../../../shared/globals.dart';
+import '../../../../shared/primitives/simple_items.dart';
 import 'tracing_data.dart';
 
 class TracingPaneController extends DisposableController
     with AutoDisposeControllerMixin {
+  TracingPaneController(this.mode);
```

Comment:
this can be const

Reply:
This constructor can't be declared 'const' because a mixin adds the instance field: 'AutoDisposeControllerMixin._delegate'.